daemon/src/inlein/daemon/read_script.clj

```diff
@@ -125,16 +125,39 @@
       (assoc-nonempty :jvm-opts (cat-merge :jvm-opts params))
       (assoc-nonempty :exclusions (cat-merge :exclusions (reverse params)))))
 
+(defn- convert-maven-dep
```

Comment:
I would like to propose a small addition to `convert-maven-dep here`: Adding in the following clause

```clojure
  ([artifact]
   (convert-maven-dep nil artifact nil nil "LATEST"))
```

In that way, we can do e.g. `inlein --repl --deps clj-time` and get the latest version of clj-time without guessing what's the latest version (or putting in LATEST manually).

Reply:
I like this, it's just a shame that it only works for dependencies where the group and artifact are the same. It would be nice to be able to say e.g. `org.clojure:clojure` and also get LATEST.
Would it be reasonable to check if the second string starts with a digit (or is equal to LATEST or possibly other predefined strings), and assume if it does it's artifact:version, and if not it's group:artifact? The user could always override it by explicitly specifying the group or LATEST if that assumption is wrong.

Reply:
Yeah, that's a good point. I think that's a good idea, and since the user can add version in cases where it doesn't work, I'm all for it.